src/ucp/api/ucp.h

```diff
@@ -262,31 +262,33 @@ enum ucp_ep_params_field {
  * @ref ucp_ep_create() function.
  */
 enum ucp_ep_params_flags_field {
-    UCP_EP_PARAMS_FLAGS_CLIENT_SERVER  = UCS_BIT(0),  /**< Using a client-server
-                                                           connection establishment
-                                                           mechanism.
-                                                           @ref ucs_sock_addr_t
-                                                           sockaddr field
-                                                           must be provided and
-                                                           contain the address
-                                                           of the remote peer */
-    UCP_EP_PARAMS_FLAGS_NO_LOOPBACK    = UCS_BIT(1),  /**< Avoid connecting the
-                                                           endpoint to itself when
-                                                           connecting the endpoint
-                                                           to the same worker it
-                                                           was created on.
-                                                           Affects protocols which
-                                                           send to a particular
-                                                           remote endpoint, for
-                                                           example stream */
-    UCP_EP_PARAMS_FLAGS_SEND_CLIENT_ID = UCS_BIT(2)   /**< Send client id
-                                                           when connecting to remote
-                                                           socket address as part of the
-                                                           connection request payload.
-                                                           On the remote side value
-                                                           can be obtained from
-                                                           @ref ucp_conn_request_h using
-                                                           @ref ucp_conn_request_query */
+    /**
+     * Using a client-server connection establishment mechanism.
+     * @ref ucs_sock_addr_t sockaddr field must be provided and contain the
+     * address of the remote peer.
+     */
+    UCP_EP_PARAMS_FLAGS_CLIENT_SERVER  = UCS_BIT(0),
+
+    /**
+     * Avoid connecting the endpoint to itself when connecting the endpoint
+     * to the same worker it was created on. Affects protocols which send to
+     * a particular remote endpoint, for example stream.
+     */
+    UCP_EP_PARAMS_FLAGS_NO_LOOPBACK    = UCS_BIT(1),
+
+    /**
+     * Send client id when connecting to remote socket address as part of the
```

Comment:
id -> identifier, better to put the full word in the comment(ary)

Reply:
done